scripts/dev

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+go run main.go "$@"
```

Comment:
@fadeev I didn't add this to the `Makefile`, I prefer small scripts that does only one job instead of big Makefiles. We can think of this like microservices vs monolithic apps. Lets discuss this if needed to create a common style guideline on it.